components/object-property-list/object-property-list.js

```diff
@@ -30,16 +31,28 @@ class ObjectPropertyList extends LocalizeCoreElement(SkeletonMixin(LitElement))
 			::slotted(:last-child), slot :last-child {
 				--d2l-object-property-list-item-separator-display: none;
 			}
+			::slotted([slot="status"]) {
+				display: none;
+			}
+			::slotted(d2l-status-indicator[slot="status"]) {
```

Comment:
I think in the readme it mentions that the slot is for a **single** status indicator. Is that true, and something we want to enforce? We might be able to lock this down further by using a selector like `::slotted(d2l-status-indicator[slot="status"]:first-of-type)`. Using `:first-child` is also an option, but would require users place the status indicator first, which is perhaps a bit too heavy-handed.

Reply:
Sure thing, done.